README.md

```diff
@@ -520,12 +520,12 @@ Uploadcare::Group.delete(group.id)
 
 # Returns group's CDN URL
 @group.cdn_url
-# => "https://ucarecdn.com/group-id~2/"
+# => "https://demo.ucarecd.net/group-id~2/"
 # With subdomains: "https://a1b2c3d4e5.ucarecdn.net/group-id~2/"
 
 # Returns CDN URLs of all files from group without API requesting
 @group.file_cdn_urls
-# => 'https://ucarecdn.com/0513dda0-582f-447d-846f-096e5df9e2bb~2/nth/0/'
+# => 'https://demo.ucarecd.net/0513dda0-582f-447d-846f-096e5df9e2bb~2/nth/0/'
 # # With subdomains: 'https://a1b2c3d4e5.ucarecdn.net/0513dda0-582f-447d-846f-096e5df9e2bb~2/nth/0/'
```

Comment:
We should remove "with subdomains" part, because now it's with subdomains by default